### ISSUE TITLE
varnish woocommerce documention

### ISSUE THREAD

**Issue by aelsharawi** [carlalexander/debops-wordpress#74]
your varnish setup misses the wocommerce cookie settings:

here 

https://docs.woothemes.com/document/configuring-caching-plugins/

**Comment by carlalexander** [comment]
Good catch! I noticed a small issue with the `varnish` role so I can't update the wiki right away.
